remoteworkitem/trackeritem_repository.go

```diff
@@ -67,6 +68,24 @@ func bindAssignees(db *gorm.DB, remoteWorkItem RemoteWorkItem, providerType stri
 	}
 	// copy all fields from remoteworkitem into result workitem
 	for fieldName, fieldValue := range remoteWorkItem.Fields {
+		// creator
+		if fieldName == remoteCreatorLogin {
+			if fieldValue == nil {
+				workItem.Fields[workitem.SystemCreator] = nil
+				continue
+			}
+			creatorLogin := fieldValue.(string)
+			creatorProfileURL := remoteWorkItem.Fields[remoteCreatorProfileURL].(string)
+			identity, err := identityRepository.Lookup(context.Background(), creatorLogin, creatorProfileURL, providerType)
+			if err != nil {
+				return nil, err
+			}
+			// associate the identities to the work item
+			workItem.Fields[workitem.SystemCreator] = identity.ID.String()
+		} else if fieldName == remoteCreatorProfileURL {
```

Comment:
because I want to ignore this field, as mentioned in the comment in the block: when processing the `remoteCreatorLogin` field, we'll also use the value of the sibling `remoteCreatorProfileURL` so we can ignore this later field later. Does it make sense ?